draft-ietf-quic-qpack.md

```diff
@@ -1047,7 +1047,7 @@ dynamic table.
 ### Indexed Field Line With Post-Base Index
 
 An indexed field line with post-base index representation identifies an entry
-in the dynamic table with an absolute index greater than or equal to the value
+in the dynamic table with an post base index greater than or equal to the value
```

Comment:
Same here -- it makes no sense to say that the post-base index is greater than or equal to the value of the Base.